website/docs/dbt-cli/install/from-source.md

```diff
@@ -19,17 +20,17 @@ pip install -r requirements.txt
 
 This will install `dbt-core` and `dbt-postgres`. To install in editable mode (includes your local changes as you make them), use `pip install -e editable-requirements.txt` instead.
 
-### Plugin
+### Installing adapter plugins
 
-To install an adapter plugin from source, first locate its repository. For instance, the `dbt-redshift` adapter is located at https://github.com/dbt-labs/dbt-redshift.git, so I can clone it and install from there:
+To install an adapter plugin from source, you will need to first locate its source repository. For instance, the `dbt-redshift` adapter is located at https://github.com/dbt-labs/dbt-redshift.git, so I can clone it and install from there:
 
 ```shell
 git clone https://github.com/dbt-labs/dbt-redshift.git
 cd dbt-redshift
 pip install .
 ```
 
-You do _not_ need to install `dbt-core` before installing an adapter plugin—it includes `dbt-core` among its dependencies, and it will install the latest compatible version automatically.
+You do _not_ need to install `dbt-core` before installing an adapter plugin -- it includes `dbt-core` among its dependencies, and it will install the latest compatible version automatically.
```

Comment:
```suggestion
You do _not_ need to install `dbt-core` before installing an adapter plugin -- the plugin includes `dbt-core` among its dependencies, and it will install the latest compatible version automatically.
```